Singer.API/Services/DatabaseService.cs

```diff
@@ -73,40 +74,6 @@ protected DatabaseService(ApplicationDbContext context, IMapper mapper)
 
       protected RoleManager<User> RoleManager { get; }
 
-      /// <summary>
```

Comment:
I guess you removed the expressions because they did not work properly, do you also know why they didn't work? Because when you remove them, we loose a lot of flexibility in the methods...

Reply:
Well, we don't need the expressions anymore. As long as we use Automapper for the obvious/ typical abstraction usage, there's no problem.